aptos-move/framework/aptos-framework/sources/block.spec.move

```diff
@@ -83,7 +85,10 @@ spec aptos_framework::block {
         aborts_if epoch_interval_microsecs <= 0;
         aborts_if exists<BlockResource>(addr);
         ensures exists<BlockResource>(addr);
+
+        // During the module's initialization, it guarantees that the BlockResource resource move under the Aptos framework account with initial values.
         ensures global<BlockResource>(addr).height == 0;
+        ensures global<BlockResource>(addr).epoch_interval > 0;
```

Comment:
same here, why don't you just use the data invariant above? Is there any issue with it?